apps/comps/lib/api-client.ts

```diff
@@ -682,11 +687,23 @@ export class ApiClient {
 
     return {
       ...res,
-      agentTotal: BigInt(res.agentTotal),
+      agentTotal: attoBoostStringToBoost(res.agentTotal),
     };
   }
 }
 
+function attoBoostStringToBoost(attoBoost: string) {
+  const res = dnum.div(attoBoost, 10 ** 18, {
+    rounding: "ROUND_DOWN",
```

Comment:
Important to round down here so we don't ever let the user boost an amount greater than what their balance is.